xml/admin_backup.xml

```diff
@@ -0,0 +1,394 @@
+     <para>
+      The Worker nodes are not affected
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      The &admin_node; is not affected
+     </para>
+    </listitem>
+   </itemizedlist>
+
+   <important>
+    <title><literal>etcd</literal> Fault Tolerance</title>
+    <para>
+     Recovering from loss of <literal>etcd</literal> Master nodes beyond
+     the fault tolerance is not in scope for this document. If you have lost
+     quorum and have surpassed fault tolerance, please contact SUSE Support.
+    </para>
+    <para>
+     <literal>etcd</literal> quorum and fault tolerance limits are described in
+     <xref linkend="sec.deploy.requirements.system.cluster.etcd_cluster_size"/>.
+    </para>
+   </important>
+
+   <para>
+    &productname; will consider failed Master nodes as "unreachable" but still
+    counting towards quorum. You must remove the failed nodes without losing the
+    quorum majority.
+   </para>
+
+   <para>
+    As long as you are within the fault tolerance. You must "force remove" the
+    failed nodes through &dashboard; and add new Master nodes until you reach
+    your desired number of master nodes.
+   </para>
+   <para>
+    The <xref linkend="sec.admin.backup.general.restore"/> is the same as for Worker nodes but
+    you must consider quorum in any case.
+   </para>
+  </sect2>
+
+  <sect2 xml:id="sec.admin.backup.general.worker">
+   <title>Worker Node Failure</title>
+   <para>
+    We assume:
+   </para>
+
+   <itemizedlist>
+    <listitem>
+     <para>
+      One or more Worker node has failed and is unrecoverable
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        Kubernetes will schedule the workloads that were running on it on another node
+       </para>
+       </listitem>
+       <listitem>
+        <para>
+         There is enough free capacity in the cluster to take existing workloads
+        </para>
+        </listitem>
+      </itemizedlist>
+    </listitem>
+    <listitem>
+     <para>
+      Workloads must shift to other available nodes during this process.
+     </para>
+    </listitem>
+   </itemizedlist>
+   <para>
+    Worker nodes should be treated as disposable. &productname; will schedule
+    workloads across the other available nodes.
+   </para>
+  </sect2>
+
+  <sect2 xml:id="sec.admin.backup.general.restore">
+   <title>Restore Procedure</title>
+   <para>
+    The restoration procedure does not concern itself with data stored on the
+    actual Master or Worker node. You will be forcing the broken nodes out of
+    the cluster and adding full replacements.
+   </para>
+   <procedure>
+    <step>
+     <para>
+     Remove the failed node: <xref linkend="sec.admin.nodes.remove" />
+     </para>
+    </step>
+    <step>
+     <para>
+     Verify the node is removed from the node list in Velum
+     </para>
+    </step>
+    <step>
+     <para>
+      Add a new master/worker node: <xref linkend="sec.deploy.nodes.worker_install" />
```

Comment:
The link here is not generated, it is only text formatted as italic font. The link above is working.

See https://w3.nue.suse.com/~mnapp/2019-02-15/book.caasp.admin/cha.admin.backup.html#sec.admin.backup.general.restore

Reply:
Another problem in this step is that the node(s) have to be upgraded to the same versions of packages as other node. Do we have somewhere described the procedure how to do manual upgrade of the node?

Reply:
No this is a quirk of Docbook and the way it handles cross references. It will `know` where the reference in the other file is but since it doesn't know where the URL is, it will spell out how to find it. There is no good way to fix this without hardcoding all links as of now. I will look into how this can be improved.